.projenrc.js

```diff
@@ -20,15 +16,15 @@ const project = new AwsCdkConstructLibrary({
   defaultReleaseBranch: "main",
   releaseEveryCommit: false,
   cdkDependenciesAsDeps: false,
+
+  publishToPypi: {
+    distName: "datadog-cdk-constructs",
+    module: "datadog-cdk-constructs",
+  },
```

Comment:
this tells JSII to generate a Python artifact with the name `datadog-cdk-constructs`

Reply:
The package name on NPM is `datadog-cdk-constructs` while the name on PyPi will be `datadog_cdk_constructs`